src/js/background/background.js

```diff
@@ -367,6 +392,14 @@ browser.runtime.onMessage.addListener(async (m, sender, _sendResponse) => {
     case "iframeCloseRelayInPageMenu":
       browser.tabs.sendMessage(sender.tab.id, {message: "iframeCloseRelayInPageMenu"});
       break;
+    case "fetchApiRequest": 
+      response = await fetchApiRequest(
+        m.url, 
+        m.fetchMethod,
+        m.body,
+        m.otps
```

Comment:
Question: I didn't run into any issues but is there any danger passing null/undefined values here? (Most of the time when this message is called, `fetchMethod`, `body` and `opts` are blank.